ynput/ayon-maya
# Disallow empty `cbId` value to pass through

### Issue

If a `cbId` attribute existed on a node, but was an empty value (e.g. user cleared it; I'm looking at you @LiborBatek ) then it would magically pass validation but break in collecting looks.

### Solution

Do not allow empty values to pass through.

### Testing notes

1. Create a model
2. Save scene
3. go to the `cbId` attribute in attribute editor, and clear the value (make it an empty string)
4. Publish the model.

The model should be invalidated and the id be regenerated.
